history/titles/c_duskwither_ravine.txt

```diff
@@ -7,4 +7,14 @@
 250.1.1={
 	holder = 42011
 }
-350.1.1={holder=310270} #Severnaya[patron]
\ No newline at end of file
+350.1.1={holder=310270} #Severnaya[patron]
+603.9.1={
+    effect = {
+        location = {
+			add_province_modifier = { name = depopulated_3 duration = -1 }
+		}
+	}
+}
+604.1.1={
+	liege=k_quelthalas
```

Comment:
but already has `liege=e_quelthalas` above?

Reply:
Fixed in 8d5699cfab69b950f50bdbd82ed471e9959d6ff9.